src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -66,10 +66,18 @@ public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusCont
                             WorkerCount = context.WorkerCount,
                             Metrics = metrics
                         };
-                        var result = monitor.GetScaleStatus(scaleStatusContext);
-
-                        _logger.LogDebug($"Monitor '{monitor.Descriptor.Id}' voted '{result.Vote.ToString()}'");
-                        votes.Add(result.Vote);
+                        var scaleStatus = monitor.GetScaleStatus(scaleStatusContext);
+                        int vote = 0;
```

Comment:
Null-coalesce would make this more concise:
```suggestion
                        int vote = scaleStatus.TargetWorkerCount ?? Convert.ToInt32(scaleStatus.Vote);
```